### ISSUE TITLE
Averages are very skewed in some categories

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#3682]
Hypothesis: 

- the kcal/kj conversion
- the lack of kcal values skewing averages ?
![image](https://user-images.githubusercontent.com/1689815/85128683-134ace80-b232-11ea-92c0-a71c109ad7e3.png)
https://world.openfoodfacts.org/product/5400119531986/trempette-a-la-tomate-delhaize

**Comment by hangy** [comment]
https://drive.google.com/file/d/1bVPe1NNwjKOte8L2q-ZL55uNURjZ3sX7/view?usp=sharing

Since only the field `energy_100g` exists for all products, maybe the individual `energy-kj_100g` and `energy-kcal_100g` fields should be ignored for averages? While the calculation might not be 100% accurate, that sounds more reasonable than comparing averages of products that don't contain explicit values for some of those fields.

**Comment by stephanegigandet** [comment]
Agreed, we could just not show comparisons for energy-kcal and energy-kj.
